Makefile

```diff
@@ -0,0 +1,103 @@
+APP_NAME = mobile-security-service-server
+ORG_NAME = aerogear
+PKG = github.com/$(ORG_NAME)/$(APP_NAME)
+APP_FILE=./cmd/mobile-security-service-server/main.go
+TOP_SRC_DIRS = pkg
+PACKAGES     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
+                   -exec dirname {} \\; | sort | uniq")
+BIN_DIR := $(GOPATH)/bin				   
+BINARY ?= mobile-security-service-server
+
+# This follows the output format for goreleaser
+BINARY_LINUX_64 = ./dist/linux_amd64/$(BINARY)
+
+DOCKER_LATEST_TAG = $(ORG_NAME)/$(APP_NAME):latest
+DOCKER_MASTER_TAG = $(ORG_NAME)/$(APP_NAME):master
+RELEASE_TAG ?= $(CIRLE_TAG)
+DOCKER_RELEASE_TAG = $(ORG_NAME)/$(APP_NAME):$(RELEASE_TAG)
+
+LDFLAGS=-ldflags "-w -s -X main.Version=${TAG}"
+
+.PHONY: setup
+setup:
+	dep ensure
+
+.PHONY: build
+build: setup
+	go build -o $(BINARY) $(APP_FILE)
+
+.PHONY: build_linux
+build_linux: setup
+	env GOOS=linux GOARCH=amd64 go build -o $(BINARY_LINUX_64) $(APP_FILE)
+
+.PHONY: docker_build
+docker_build: build_linux
+	docker build -t $(DOCKER_LATEST_TAG) --build-arg BINARY=$(BINARY_LINUX_64) .
+
+.PHONY: docker_build_release
+docker_build_release: build_linux
+	docker build -t $(DOCKER_RELEASE_TAG) --build-arg BINARY=$(BINARY_LINUX_64) .
+
+.PHONY: docker_build_master
+docker_build_master: build_linux
+	docker build -t $(DOCKER_MASTER_TAG) --build-arg BINARY=$(BINARY_LINUX_64) .
+
+.PHONY: test
+test: test-unit
```

Comment:
I don't see this documented - unintentionally omitted?

Reply:
@damienomurchu This is included in the table. See:

| Command                       | Description                                                                                     |
|-------------------------------|-------------------------------------------------------------------------------------------------
| `make test`                   | Runs unit tests                                                                                 |